cmd/dr/checks.go

```diff
@@ -35,8 +35,7 @@ var AllConfigChecks = []Check{
 
 // FileChecks defines the checks that are done for files on the system.
 var FileChecks = []Check{
-	NewBinaryFileCheck("podman", "%s was not found on your path", UpdateConfig("podman.path")),
-	NewBinaryFileCheck("docker", "%s was not found on your path", UpdateConfigIfMissing("podman.path")),
+	NewResourceFileCheck(OneExistsOnPath([]string{"podman","docker"}), "%s was not found on your path", UpdateConfig("podman.path")),
```

Comment:
If you use the syntax sugar that I have below, you can use this:

```
NewResourceFileCheck(OneExistsOnPath("podman","docker"), "%s was not found on your path", UpdateConfig("podman.path")),
```

Reply:
added this